tests/playwright/sanity/modules/checklist/helper.ts

```diff
@@ -78,28 +78,29 @@ export default class ChecklistHelper {
 				title: 'Create your first 3 pages',
 				imagePath: 'https://assets.elementor.com/checklist/v1/images/checklist-step-3.jpg',
 				description: 'Jumpstart your creation with professional designs form the Template Library or start from scratch.',
-				link: 'https://elementor.com/help/create-new-page/',
-				CTA: 'Create a new page',
-				isCompleted: true,
-			},
-			{
-				id: 'add-header',
-				title: 'Create your first 3 pages',
-				imagePath: 'https://assets.elementor.com/checklist/v1/images/checklist-step-4.jpg',
-				description: 'Jumpstart your creation with professional designs form the Template Library or start from scratch.',
-				link: 'https://elementor.com/help/create-new-page/',
-				CTA: 'Create a new page',
-				isCompleted: false,
-			},
-			{
-				id: 'add-404',
-				title: 'Create your first 3 pages',
-				imagePath: 'https://assets.elementor.com/checklist/v1/images/checklist-step-5.jpg',
-				description: 'Jumpstart your creation with professional designs form the Template Library or start from scratch.',
-				link: 'https://elementor.com/help/create-new-page/',
-				CTA: 'Create a new page',
-				isCompleted: false,
+				learn_more_link: 'https://elementor.com/help/create-new-page/',
+				learn_more_text: 'https://elementor.com/help/create-new-page/',
+				cta_text: 'Create a new page',
+				is_completed: true,
 			},
+			// {
+			// 	id: 'add-header',
+			// 	title: 'Create your first 3 pages',
+			// 	imagePath: 'https://assets.elementor.com/checklist/v1/images/checklist-step-4.jpg',
+			// 	description: 'Jumpstart your creation with professional designs form the Template Library or start from scratch.',
+			// 	link: 'https://elementor.com/help/create-new-page/',
+			// 	CTA: 'Create a new page',
+			// 	isCompleted: false,
+			// },
+			// {
+			// 	id: 'add-404',
+			// 	title: 'Create your first 3 pages',
+			// 	imagePath: 'https://assets.elementor.com/checklist/v1/images/checklist-step-5.jpg',
+			// 	description: 'Jumpstart your creation with professional designs form the Template Library or start from scratch.',
+			// 	link: 'https://elementor.com/help/create-new-page/',
+			// 	CTA: 'Create a new page',
+			// 	isCompleted: false,
+			// },
```

Comment:
Since all tests are passing we no longer need to save it
so it can also be removed